Refit/MultipartItem.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Refit
+{
+    public abstract class MultipartItem
+    {
+        public MultipartItem()
+        {
+
+        }
+
+        public MultipartItem(string fileName, string contentType)
+        {
+            this.FileName = fileName;
+            this.ContentType = contentType;
+        }
+
+        public virtual string FileName { get; set; }
+        public virtual string ContentType { get; set; }
+
+        public virtual HttpContent ToContent()
+        {
+            var content = CreateContent();
+            if (!string.IsNullOrEmpty(ContentType))
```

Comment:
Given that ContentType wasn't required previously, I imagine there are web services out there that don't require it.